lib/cylc/gui/option_group.py

```diff
@@ -87,11 +87,11 @@ def get_options(self):
         else:
             options = ' '
         for name in self.entries:
-            (entry, label, option) = self.entries[name]
+            (entry, _, option) = self.entries[name]
             if entry.get_text():
                 options += ' ' + option + entry.get_text()
         for name in self.arg_entries:
-            (entry, label) = self.arg_entries[name]
+            entry = self.arg_entries[name][0]
             if entry.get_text():
                 options += ' ' + entry.get_text()
```

Comment:
`name` used once as index to `self.entries` in the the loops. Perhaps we should loop `self.entries.values()` instead?

Reply:
Done.